packages/twenty-front/src/modules/object-record/record-show/hooks/useFindRecordCursorFromFindManyCacheRootQuery.ts

```diff
@@ -0,0 +1,34 @@
+import { RecordGqlRefEdge } from '@/object-record/cache/types/RecordGqlRefEdge';
+import { useApolloClient } from '@apollo/client';
+import { createApolloStoreFieldName } from '~/utils/createApolloStoreFieldName';
+
+export const useFindRecordCursorFromFindManyCacheRootQuery = ({
```

Comment:
couldn't we have two queries with same fieldVariables? I think we have no other choice than to pass the cursor in the url / store it in a recoil state

Reply:
I would use a recoilState